lib/model/Resource.js

```diff
@@ -0,0 +1,123 @@
+/** @ignore @typedef {import('./WheelSlot.js').default} WheelSlot */
+
+/**
+ * Information about a resource.
+ */
+class Resource {
+  /**
+   * Creates a new Resource instance.
+   * @param {Object|null} jsonObject A embedded resource object from the fixture's JSON data.
+   * @param {WheelSlot} wheelSlot The wheel slot that this resource belongs to.
+   */
+  constructor(jsonObject, wheelSlot) {
+    this._jsonObject = jsonObject;
+    this._wheelSlot = wheelSlot;
```

Comment:
This is not needed, isn't it? It seems a bit odd to make the Resource object dependent on the WheelSlot.

(Of course, the Resource object is newly created for each WheelSlot, even if multiple slots refer to the same resource. So the connection WheelSlot → Resource does actually exist, but I don't think it's what we want. In future, a Resource caching solution or something like that would be the desirable option.)